docusaurus/docs/React/components/contexts/thread-context.mdx

```diff
@@ -7,9 +7,9 @@ title: ThreadContext
 
 ## ThreadProvider
```

Comment:
I don't think we should document `ThreadProvider` - for the same reason we don't document any other providers, they are not user-facing. It's better to add a few more details about `ThreadAdapter` instead.

Reply:
`ThreadAdapter` is a primarily `ChatView` component. It utilises context of the `ChatView.Threads` which provides setter for the `ThreadList` and a "selected thread" value for the `useActiveThread` (handles thread activation based on document visibility and focus states). So for custom purposes such as [this one](https://github.com/GetStream/stream-chat-react/blob/58b22fa9f9175ab5bb165f3f2f77d2497ba45b0d/docusaurus/docs/React/guides/custom-threads-view.mdx#building-custom-threads-view) it becomes unnecessary since the whole "get/set" behavior is being replaced.